src/components/WatchlistEdit/AssetsList.js

```diff
@@ -35,7 +42,7 @@ const AssetsList = ({ items, listItems, isContained, onToggleProject }) => {
   }
 
   const wrapperStyles = {
-    height: items.length > 4 ? '145px' : `${32 * items.length}px`,
+    height: items.length > 4 ? `${height || 145}px` : `${32 * items.length}px`,
```

Comment:
why did you change `height`? `145px` used for correct height with overflow styles:
![image](https://user-images.githubusercontent.com/24521041/61170019-4e497a00-a56c-11e9-9735-9d4017049dcc.png)
in you solution now scroll change his position, looks not so nice.
and as I remember, Andrey decided to use checkboxes here. You can see it in figma:
![image](https://user-images.githubusercontent.com/24521041/61170172-c7959c80-a56d-11e9-861b-b89c4509b982.png)
component for this [here](https://github.com/santiment/app/blob/master/src/components/WatchlistCopy/AssetsList.js)
with `classes.list` you can pass your height as [here](https://github.com/santiment/app/blob/master/src/components/WatchlistCopy/WatchlistCopyPopup.module.scss#L80)